prov/mrail/src/mrail_cq.c

```diff
@@ -437,7 +437,7 @@ static void mrail_cq_progress(struct util_cq *cq)
 	mrail_poll_cq(cq);
 
 	/* Progress the bound EPs */
-	ofi_cq_progress(cq);
+	ofi_cq_progress_ts(cq);
```

Comment:
I think, it would be great to have ts and non-ts version.
If you don't agree with me, please, ignore my comment about `fastlock_acquire/release`?

Reply:
We do have separate versions with this change.  The non-ts version just isn't exported yet, since there's no user.  I used the ts version of the call here simply because that's what the original code did, and I didn't want to spend the time figuring out if a non-ts version is okay.  For example, with the tcp provider, using a non-ts implementation results in a race condition